extensions/ql-vscode/src/run-queries.ts

```diff
@@ -267,14 +277,21 @@ export class QueryEvaluationInfo {
         pageSize: 100,
         offset: nextOffset,
       });
-      for (const tuple of chunk.tuples)
+      for (const tuple of chunk.tuples) {
         out.write(tuple.join(',') + '\n');
+      }
       nextOffset = chunk.next;
     }
     out.end();
   }
 
-  async ensureCsvProduced(qs: qsClient.QueryServerClient, dbm: DatabaseManager): Promise<string> {
+  /**
+   * Returns the path to the CSV alerts interpretation of this query results. If CSV results have
+   * not yet been produced, this will return first create the CSV results and then return the path.
+   *
+   * This method only works for queries with interpreted results.
```

Comment:
What does it do for queries without interpreted results?  Does it throw?

Reply:
Good question. I'm not really sure. It is currently set up so that this is only called when there are interpreted results. I was just trying to clarify my own understanding by adding these comments. Let me see what it does...

...yes. It throws from inside the CLI.